docs/content/v2.16/architecture/transactions/read-committed.md

```diff
@@ -323,7 +323,7 @@ This can change after [#11573](https://github.com/yugabyte/yugabyte-db/issues/11
 
 ### Interaction with concurrency control
 
-Semantics of Read Committed isolation adheres only with the [Wait-on-Conflict](../concurrency-control/#wait-on-conflict-beta-preview-faq-general-what-is-the-definition-of-the-beta-feature-tag) concurrency control policy. This is because a Read Committed transaction has to wait for other transactions to commit or rollback in case of a conflict, and then perform the re-check steps to make progress.
+Semantics of Read Committed isolation adheres only with the [Wait-on-Conflict](../concurrency-control/#wait-on-conflict) concurrency control policy. This is because a Read Committed transaction has to wait for other transactions to commit or rollback in case of a conflict, and then perform the re-check steps to make progress.
```

Comment:
concurrency-control page for the wait and fail on conflict links doesn't appear to exist in 2.16 - not sure if it is simply missing, or what - none of the other pages refer to concurrency-control in 2.16, so maybe this is here incorrectly?

Reply:
Filing separate ticket for this issue